vitrivr-engine-core/src/main/kotlin/org/vitrivr/engine/core/model/content/impl/cache/CachedTextContent.kt

```diff
@@ -14,6 +15,8 @@ import java.nio.file.StandardOpenOption
  */
 class CachedTextContent(override val path: Path, text: String) : TextContent, CachedContent {
 
+    override val id: UUID = UUID.randomUUID()
```

Comment:
I would add this to the constructor. There might be use-cases, where one wants to define this explicitly. You can also use default values in the constructor.